### PR TITLE
Internationalization - Facet Category/Facet Label

### PR DESCRIPTION
**Pull request by JayanthyChengan**
For both facet categories and Facet labels, key/value pairs are added to bundle files based on what solr returns and fetched into UI ( search-include-fragment.xhtml )

## Related Issues

- connects to #5207

### CODE REVIEW

File: src/main/java/Bundle_fr.properties [hunk 1]
```diff
@@ -8,8 +8,8 @@ datasets=Ensembles de donn
 newDataset=Nouvel ensemble de donn�es
```
Comment (pdurbin) [inline src/main/java/Bundle_fr.properties]: When will this entire Bundle_fr.properties file be removed? It's the plan for installations who want French support to get the Bundle_fr.properties file from https://github.com/GlobalDataverseCommunityConsortium/dataverse-language-packs ?

File: src/main/java/Bundle.properties [hunk 2]
```diff
@@ -2289,3 +2289,932 @@ rtabfileparser.ioexception.boolean=Unexpected value for the Boolean variable ({0
...
+Organization\u0020or\u0020Institution=Organization or Institution
+Teaching\u0020Course=Teaching Course
+Research\u0020Group=Research Group
+Laboratory=Laboratory
+Department=Department
+Uncategorized=Uncategorized
+
+#astrophysics
+Image=Image
+Mosaic=Mosaic
```
Comment (pdurbin) [inline src/main/java/Bundle.properties]: Do we really want strings that were formerly in astrophysics.tsv (for example) in the main Bundle.properties file? Should they go in src/main/java/astrophysics.properties instead? Also, does metadatacustomization.rst need to be updated to mention facets and where to put the strings when creating a custom metadata block?

File: src/main/java/Bundle_fr.properties (hunk 1 above)
Comment (JayanthyChengan) [reply src/main/java/Bundle_fr.properties]: @pdurbin - Yes, I would like to create a separate issue to remove all french language files and Bundle_zh_CN.properties from codebase.  Also to move all the English property files into a specific folder.  Eg:  under src/main/java/english/*.properties , so this folder can be synchronized automatically (through Jenkins setup) to the English folder in dataverse-language-packs repository.  Later, translators can work for other languages.  Please add your comments.  Thanks

File: src/main/java/Bundle.properties (hunk 2 above)
Comment (JayanthyChengan) [reply src/main/java/Bundle.properties]: @pdurbin - The key added to the bundle.properties are the values returned from solr for facets.  Is there any way to identify which metadata property file it corresponds to?

**Comment by scolapasta** [comment]
closing in favor of #5697

**Pull request closed without merging**